### PR TITLE
add landing page and dashboard outline

### PR DESCRIPTION
**Pull request by yujinjcho**

### CODE REVIEW

File: src/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import Container from 'react-bootstrap/lib/Container';
+
+class LandingPage extends Component {
```
Comment (achiu8) [inline src/LandingPage.js]: Will this component have state? Why `class` for this vs. pure function for `Dashboard`?

File: src/Home.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-export default withUserValidation(Home);
+class Home extends Component {
+
+  state = {
+    loggedIn: auth.loggedIn()
+  };
+
+  render() {
+    const { loggedIn } = this.state;
+    return loggedIn ? < Dashboard /> : < LandingPage />
```
Comment (achiu8) [inline src/Home.js]: Nit: shouldn't have space after opening brackets.

File: src/SignOutLink.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Nav } from 'react-bootstrap';
+import auth from './auth';
+
+class SignOutLink extends Component {
+
+  logOut() {
```
Comment (achiu8) [inline src/SignOutLink.js]: Typically prefix event handler names with "handle".

File: src/Navigation.js [hunk 4]
```diff
@@ -10,11 +12,16 @@ class Navigation extends Component {
...
+        <Navbar.Brand>
+          <Link to='/' >Rcruitme</Link>
+        </Navbar.Brand>
         <Nav className="mr-auto">
         </Nav>
+
+
         <Nav>
-          {auth.loggedIn() ? <SignOut /> : <CreateAccount />}
+          { !auth.loggedIn() && <CreateAccount /> }
```
Comment (achiu8) [inline src/Navigation.js]: Just a note: may potentially have to rethink how we're handling / keeping track of logged in state at some point. Currently, changes not accounted for in render lifecycle.

Comment (yujinjcho) [reply src/Navigation.js]: As in since we're using local storage if that changes, that would not necessarily cause this component to reupdate?

Comment (achiu8) [reply src/Navigation.js]: Ya

**Pull request merged**